### PR TITLE
Withdrawals to return ready as well as pending withdrawals

### PR DESCRIPTION
**Pull request by AliMeer**
This updates getWithdrawals to return withdrawal requests which are ready as well as the ones which are not ready.
The withdrawal requests which are ready will show a total amount available for withdrawal clubbed together by token.
The withdrawal requests which are not ready will show individual withdrawal requests which have been made by the user(eg. ETH 1, ETH 5, ETH 9)

### CODE REVIEW

File: src/api/contract/getStarkKey.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const DVFError = require('../../lib/dvf/DVFError')
+const BN = require('bignumber.js')
+
+module.exports = async dvf => {
+  const ethAddress = dvf.get('account')
+  const args = [ethAddress]
+  try {
+    const sarkKeyDecimal = await dvf.eth.call(
```
Comment (plutoegg) [inline src/api/contract/getStarkKey.js]: 'sarkKey' => starkKey

Comment (AliMeer) [reply src/api/contract/getStarkKey.js]: done

**Comment by plutoegg** [comment]
@AliMeer i believe this needs to be updated to use /r/getPendingWithdrawals API, as well as test cases before merged

**Comment by AliMeer** [comment]
> @AliMeer i believe this needs to be updated to use /r/getPendingWithdrawals API, as well as test cases before merged

This has now been updated.

**Pull request merged**